README.md

```diff
@@ -1,6 +1,10 @@
 # Ruby Enumerable methods
 A comprehensive rebuild on basic Enumerable methods in Ruby for a better understanding in block passing, enumerator and classes and regular expressions as arguments.
 
+## Update: Added Rspec testing for all methods, check ./spec/enumerable_spec.rb for details.
+
+### Testing project specification can be found at: https://www.theodinproject.com/courses/ruby-programming/lessons/testing-your-ruby-code
+
 ### This scripts rebuilds ruby Enumerable methods:
```

Comment:
- [ ] The project should have a descriptive README.md ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))